crates/wasmtime/src/instance.rs

```diff
@@ -15,33 +15,11 @@ use wasmtime_environ::wasm::{
 use wasmtime_environ::Initializer;
 use wasmtime_jit::TypeTables;
 use wasmtime_runtime::{
-    Imports, InstanceHandle, InstantiationError, StackMapRegistry, VMContext,
-    VMExternRefActivationsTable, VMFunctionBody, VMFunctionImport, VMGlobalImport, VMMemoryImport,
-    VMTableImport,
+    Imports, InstanceAllocationRequest, InstanceHandle, InstantiationError, StackMapRegistry,
+    VMContext, VMExternRefActivationsTable, VMFunctionBody, VMFunctionImport, VMGlobalImport,
+    VMMemoryImport, VMTableImport,
 };
 
-/// Performs all low-level steps necessary for instantiation.
```

Comment:
Whoops, this comment accidentally got removed during the rebase.  Adding it back now.